modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectStops.java

```diff
@@ -199,7 +199,7 @@ boolean containsStops(JSONObject jsonObject) {
             String key = (String) keys.next();
             if (jsonObject.get(key) instanceof JSONObject) {
                 JSONObject child = (JSONObject) jsonObject.get(key);
-                if (child.containsKey("stops")) {
+                if (child.containsKey("stops") && ((JSONArray)((JSONArray)child.get("stops")).get(0)).get(0) instanceof JSONObject) {
```

Comment:
That was what was in place before David Vicks last change to this class and appeared to me to have maybe broken the SLD transformation - will revisit and see if I can correct it.